File: src/MapEditor/bin/Debug/Scripts/Logic/old/Grass.lua
function Load(logic)
	logic.name = "Grass"
	logic.s_0 = 0
	logic.s_1 = 10

	logic:addSameType(0, 0, 19)
end

function Eval(logic, info, u, d, l, r, ul, ur, dl, dr)
	info:Adjust(0, 10, 0)
	r = math.random(0, 25)

	if (r == 0) then
		sprites = {}
		sprites[0] = 10
		sprites[1] = 11
		sprites[2] = 12
		sprites[3] = 23
		sprites[4] = 24
		sprites[5] = 25
		sprites[6] = 38
		sprites[7] = 62
		sprites[8] = 63
		sprites[9] = 64
		sprites[10] = 62 + 13
		sprites[11] = 63 + 13
		sprites[12] = 64 + 13
		sprites[13] = 62 + 26
		sprites[14] = 63 + 26
		sprites[15] = 64 + 26
		sprites[16] = 62 + 39
		sprites[17] = 63 + 39
		sprites[18] = 64 + 39
		info:Adjust(0, sprites[math.random(0, table.getn(sprites)) - 1])
	end

end

File: src/MapEditor/bin/Debug/Scripts/Logic/Grass.cs
using General.Script;
using MapEditor.Data.TileLogicNew;

public class Grass : IScript {

	const string tname = "Outside";
	const int x = 0;
	const int y = 0;

	public void Load(TileLogicScript logic) {
		logic.name = "Grass";

		logic.s_0 = tname;
		logic.s_1 = 0;

		logic.addSameType(tname, x, y);
		logic.addSameType(tname, x, y + 1);
		logic.addSameType(tname, x, y + 2);
		logic.addSameType(tname, x, y + 3);
		logic.addSameType(tname, x, y + 4);
	}

	public void Eval(TileLogicScript logic, LogicEvalInfo info, bool u, bool d, bool l, bool r, bool ul, bool ur, bool dl, bool dr) {
		info.Adjust(tname, x, y);
	}

}